__snapshots__/TransferVerifier.test.js

```diff
@@ -1,3 +1,3 @@
-exports['TransferVerifier tokenTransfer() gas cost 1'] = 95332
+exports['TransferVerifier tokenTransfer() gas cost 1'] = 85132
```

Comment:
why is this changing on this commit? nothing on TransferVerifier changed. probably should re-run the tests/snapshots and make sure this stays the same